src/eureka/S5_lightcurve_fitting/differentiable_models/QuasiLambertianPhaseCurve.py

```diff
@@ -110,7 +110,8 @@ def eval(self, eval=True, channel=None, pid=None, **kwargs):
                     phi = anom + pl_params.w*np.pi/180 + np.pi/2
 
                 # calculate the phase variations
-                phaseVars = abs(lib.cos((phi-self.quasi_offset*np.pi/180)/2)
+                phaseVars = abs(lib.cos(
+                                (phi+pl_params.quasi_offset*np.pi/180)/2)
```

Comment:
Can you please also add a note on the docs page that the quasi_offset is in degrees East (aka before eclipse)